ci/Jenkinsfile.qt-build

```
@@ -0,0 +1,224 @@
+                    """
+                }
+            }
+        }
+
+        stage('Qt Android') {
+            when {
+                expression { params.BUILD_QT_FROM_SOURCE == true }
+            }
+            agent {
+                docker {
+                    label 'linux && qt-builder'
+                    image "${DOCKER_REGISTRY}/qt-dev-base:noble-17-35-27.2.12479018"
+                    args "-u root -v ${WORKSPACE}/qt_export:/root/export -v ${WORKSPACE}/ci/docker/qt-build/scripts:/root/scripts"
+                }
+            }
+            steps {
+                script {
+                    echo "Building Qt ${params.QT_VERSION} for Android..."
+                    sh """
+                        chmod +x /root/scripts/build_qt_android.sh /root/scripts/build_qt_desktop.sh
+                        cd /root
+                        /root/scripts/build_qt_android.sh
+                    """
+                }
+            }
+        }
+
+        stage('Build Qt CI Image') {
+            steps {
+                script {
+                    echo "Building final Qt Docker image..."
+                    sh """
+                        cd ${QT_BUILD_DIR}/qt
+                        cp -r ${WORKSPACE}/qt_export .
+                        docker build \
+                            --build-arg QTVER=${params.QT_VERSION} \
+                            -t ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG} .
+                    """
+                }
+            }
+        }
+
+        stage('Build Enhanced Runtime Image') {
+            steps {
+                script {
+                    echo "Building enhanced runtime image with Nim and all tooling..."
+                    sh """
+                        cd ${QT_BUILD_DIR}/nim-runtime
+                        docker build \
+                            --build-arg GOLANG_VERSION=1.23.10 \
+                            --build-arg NIM_VERSION=2.0.12 \
+                            --build-arg NIX_VERSION=2.24.11 \
+                            -t ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci .
+                    """
+                }
+            }
+        }
+
+        stage('Test Images') {
+            parallel {
+                stage('Test Qt Base Image') {
+                    steps {
+                        script {
+                            echo "Testing the Qt base image..."
+                            sh """
+                                docker run --rm \
+                                    ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG} \
+                                    "qmake --version"
+
+                                docker run --rm \
+                                    ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG} \
+                                    "ls -la /opt/qt/${params.QT_VERSION}/"
+                            """
+                        }
+                    }
```

You're indenting things a lot.
```suggestion
                    steps { script {
                        image.inside('--entrypoint=""') { c ->
                            sh "qmake --version"
                            sh "ls -la /opt/qt/${params.QT_VERSION}/"
                        }
                    } }
```